scss/subLessonSlug.module.scss

```diff
@@ -1,19 +1,32 @@
+@use './colors.module.scss';
+
 .sublesson__container {
   display: grid;
-  gap: 40px;
+  gap: 1rem;
   position: relative;
   padding: 0;
 }
 
 .sublesson__sidebar {
   top: 20px;
   height: fit-content;
-  width: fit-content;
 }
 
 .toc__container {
   list-style: none;
-  background: white;
-  width: fit-content;
+  background: colors.$fg-light;
   height: fit-content;
 }
+
+.toc__header {
+  background-color: colors.$bg-white;
+}
+
+.toc__toggle {
```

Comment:
The toggle should probably have a hover effect or something to signal that the element is clickable.

Reply:
Nevermind.

Reply:
Yeah, it's not clickable on the desktop as it's always open.